### PR TITLE
Edge/cie 4916 doc updates based on internal testing feedback

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

**Review by bbyreddy — APPROVED** [review]

File: static/files/edge-k8s/c8yedge.yaml [hunk 1]
```diff
@@ -3,32 +3,34 @@
...
 
-  company: <COMPANY-NAME>                                                   # Replace with the company name
-  domain: <DOMAIN-NAME>                                                     # Replace with the domain of your choice
-  email: <ADMIN-EMAIL>                                                      # Replace with a valid email, used for admin account
+  company: <COMPANY-NAME>                                                   # Replace with the company name.
+  domain: <DOMAIN-NAME>                                                     # Replace with the domain of your choice.
+  email: <ADMIN-EMAIL>                                                      # Replace with a valid email, used for admin account.
 
-#  tlsSecretName: myown-iot-com-tls                                         # Secret for supplying the TLS Key/Certificates of Edge doamin
+#  tlsSecretName: myown-iot-com-tls                                         # Secret for supplying the TLS Key/Certificates of Edge domain. This is an optional field and if not provided the Operator will generate self-signed certificates.
```
Comment (MWindel) [inline static/files/edge-k8s/c8yedge.yaml:25]: Is "Operator" a proper name like "Edge Operator"? If so, please use the specification (Edge, Kubernetes etc.) If not, then please spell it in lowercase.

Also occurrences below

Comment (sharath-jk) [reply static/files/edge-k8s/c8yedge.yaml:25]: @MWindel - this is developer file for reference to be used as is and is authored by the engineering team. So not sure if we have to change this.

Comment (sharath-jk) [reply static/files/edge-k8s/c8yedge.yaml]: Updated to "Edge Operator" after confirming with Bhaskar.

**Comment by sharath-jk** [comment]
> @sharath-jk We probably need a second PR here for the develop branch?

@BeateRixen - This content is for Edge on Kubernetes. Once all tasks are complete for Edge on Kubernetes, we can move it to develop. Please let me know what you think.

**Comment by BeateRixen** [comment]
> > @sharath-jk We probably need a second PR here for the develop branch?
> 
> @BeateRixen - This content is for Edge on Kubernetes. Once all tasks are complete for Edge on Kubernetes, we can move it to develop. Please let me know what you think.

@sharath-jk Got it. Sure.

**Review by MWindel — APPROVED** [review]

**Pull request merged**